plaso/containers/events.py

```diff
@@ -82,6 +82,57 @@ def CalculateEventValuesHash(event_data, event_data_stream):
   return md5_context.hexdigest()
 
 
+class DateLessLogHelper(interface.AttributeContainer):
+  """Date-less log helper attribute container.
+
+  Attributes:
+    earliest_date (datetime.datetime): earliest possible date the event data
```

Comment:
why use datetime, why not use a tuple of (year, month, day) ?

Reply:
I didn't think about it. It is a better idea.

Reply:
went for tuples in https://github.com/log2timeline/plaso/pull/4854